easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb

```
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'phyx'
+version = '1.3'
+
+homepage = 'https://github.com/FePhyFoFum/%(name)s'
```

please don't use templates in the homepage url, as that makes it difficult to copy-paste into a browser.
```suggestion
homepage = 'https://github.com/FePhyFoFum/phyx'
```